kube_aws_autoscaler/main.py

```diff
@@ -46,6 +46,7 @@
 
 logger = logging.getLogger('autoscaler')
 
+ASG_ALLOWED_TAGS = {'buffer-memory-percentage': 'memory', 'buffer-cpu-percentage': 'cpu', 'buffer-pods-percentage': 'pods', 'buffer-spare-nodes': 'nodes'}
```

Comment:
Perhaps we should use a prefix on those tags to help identify their purpose at a glance.
Ex:
  aws-autoscaler-buffer-memory-percentage
  aws-autoscaler-buffer-cpu-percentage
  etc

Reply:
Added kube-aws-autoscaler: as prefix.